docs/mint.json

```diff
@@ -67,6 +67,7 @@
         {
           "group": "🔗 Integrations",
           "pages": [
+            "integration/helicone",
```

Comment:
Hey, Can you please move it to the last of this list?

Reply:
@Dev-Khant done!